### PR TITLE
Tao 875 section buttons

### PR DESCRIPTION
**Pull request by ssipasseuth**
The modification here allow adding button on a item or section basis, as requested here https://oat-sa.atlassian.net/browse/TAO-874 and here https://oat-sa.atlassian.net/browse/TAO-875

To test this, please use the sample qti test package available at https://oat-sa.atlassian.net/browse/TAO-875

### CODE REVIEW

**Comment by ssipasseuth** [comment]
THis requires https://github.com/oat-sa/extension-tao-act/pull/10

File: views/js/controller/runtime/testRunner.js [hunk 1]
```diff
@@ -288,6 +288,14 @@ define([
                     $controls.$skip.hide();
                     $controls.$skipEnd.hide();
                 }
+                
+                var config = module.config();
```
Comment (krampstudio) [inline views/js/controller/runtime/testRunner.js]: Could you move it to the top of the function

File: views/js/controller/runtime/testRunner.js [hunk 2]
```diff
@@ -288,6 +288,14 @@ define([
...
+				var $toolsContainer = $('.tools-box-list');
```
Comment (krampstudio) [inline views/js/controller/runtime/testRunner.js]: You may only need to select the element in the `if` condition.

**Comment by ssipasseuth** [comment]
suggested changes implemented

**Pull request merged**